Manager/src/main/java/io/raspberrywallet/manager/modules/authorizationserver/AuthorizationServerAPI.java

```diff
@@ -22,15 +23,17 @@
     private ApacheHttpClient httpClient;
 
     private Token token;
-    private Boolean isRegisteredFlag, isLoggedInFlag;
 
     AuthorizationServerAPI(@NotNull AuthorizationServerConfig configuration) {
         this.configuration = configuration;
         Form defaultHeaders = Form.form()
                 .add(HttpHeaders.CONTENT_TYPE, "application/json")
                 .add("charset", "UTF-8");
 
-        httpClient = new UnsecureApacheHttpClient(defaultHeaders);
+        if (configuration.getAddress().startsWith("http://"))
+            httpClient = new UnsecureApacheHttpClient(defaultHeaders);
+        else
+            httpClient = new SecureApacheHttpClient(defaultHeaders, configuration.getAcceptUntrustedCerts());
```

Comment:
HTTP is not deprected, but shouldn't be used, since it is disabled on Authorization Server.
Let's say that code works, but is disabled and may be used in the future for API, that uses HTTP

Reply:
Also I think that getAddress should always contain protocol and it's okay to even throw if it doesn't

Reply:
Me too, but descriptive exception message would be nice.

Reply:
Where it can be implemented? 

I'm thinking about setters in Config classes for specific Modules, RuntimeException would be correct, since Config must be valid all the time + I think, that you implemented validation in reading yaml configuration, where we can throw and catch IllegalArgumentException, which will be then showed to user as invalid yaml config file

Should I create Issue for this? It's not main feature, more like nice to have

Reply:
Yes, please create issue. I have idea how to solve this problem. And yes, it's nice to have.

Reply:
Done in #85